packages/collections/specs/unit/base/Sequence.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { Sequence } from '../../../..';
+import { Sequence } from '../../../index';
```

Comment:
`index` is redundant, you can omit it and rewrite import to just `../../..`

Reply:
Without `index` the path becomes completely unreadable. Dots already make it hard to understand where we are going and explicit `index` make it a bit easier to see the point of the path.
Furthermore, during the next steps, I wanted to suggest using development aliases to prevent usage of dots. Please check out [this article](https://dev.to/larswaechter/path-aliases-with-typescript-in-nodejs-4353) to get the point.